f5-openstack-agent-dist/Docker/ubuntu/install_test/Dockerfile.trusty

```diff
@@ -4,7 +4,10 @@ FROM ubuntu:trusty
 RUN apt-get update -y && apt-get install software-properties-common -y
 RUN apt-add-repository cloud-archive:liberty
 RUN apt-get update -y && apt-get dist-upgrade -y --force-yes
+RUN apt-add-repository cloud-archive:mitaka
+RUN apt-get update -y
 RUN apt-get install python-requests git curl -y --force-yes
+RUN apt-get install python-six
 
 COPY ./fetch_and_install_deps.py /
 
```

Comment:
Can do.  Will grep for where else 'ENTRYPOINT' and 'CMD' is being used in-code and try and script-kitty.  Hopefully, I'll have it working by the time you get in this afternoon.